packages/mask/src/plugins/Avatar/types.ts

```diff
@@ -51,3 +53,10 @@ export interface NFTInfo {
     slug: string
     permalink: string
 }
+
+export enum SET_NFT_FLAG {
```

Comment:
I would suggest
```suggestion
export enum NFT_USAGE {
```

or

```suggestion
export enum NFT_SHOWCASE {
```